proto/cacheclient.proto

```diff
@@ -122,6 +127,19 @@ message _DictionarySetRequest {
 
 message _DictionarySetResponse {}
 
+message _DictionaryIncrementRequest {
+  bytes dictionary_name = 1;
+  bytes field = 2;
+  int64 amount = 3;
+  uint64 ttl_milliseconds = 4;
+  bool refresh_ttl = 5;
+}
+
+message _DictionaryIncrementResponse {
+  ECacheIncrementResult status = 1;
```

Comment:
Do we need this status? I think the plan was to even deprecate the `ECacheResult` and have oneof `Hit` or `Miss` in the response. We don't return a status for any of the other apis for error cases.

Reply:
We need a way to communicate if the increment failed and why, in particular for parsing errorrs, overflow errors, and underflow errors. Do we have a preferred way to do this?

Reply:
We use status object only for valid non-error cases. For error cases we return appropriate grpc status codes with apt error messages.